docs/Parallel-Building.md

```diff
@@ -12,7 +12,7 @@ You may have been directed here by the following notice output by Theos:
 
 These instructions will help you install the latest version of GNU Make on macOS. No other modern operating system should end up in this situation, as Make 4.0 was released in 2013. If you see this notice on any other OS such as Linux, you should consider upgrading your entire OS.
 
-These instructions assume you use Homebrew, and have installed Theos as we recommend in the [official installation instructions](https://github.com/theos/theos/wiki/Installation).
+These instructions assume you use Homebrew and have installed Theos as outlined in the [official installation instructions](Installation.md).
```

Comment:
Let’s remove “official” here, it’s not really necessary in the context of this sentence.